### ISSUE TITLE
AI controlled characters not using randomized spell list

### ISSUE THREAD

**Issue by VRNocturne** [gameboy9/DW4RandoHacker#93]
I created a seed with Ludicrous hero spells setting and starting in Ch. 5 (in case that matters). I picked up Nara and Mara and went into battle, but they just used their vanilla spells. 

Nara had Blazemost and Mara had Bikill. (Hero started with Explodet lol) I figured they wouldn't be using spells at level 1 with those two as their starting spells, but instead they used their vanilla spells.

I tried changing party tactics, regenerating the seed, restarting the rom/emu, but the AI continued using vanilla spells.

**Comment by gameboy9** [comment]
Yeah this is a long standing issue.  I basically need to find the AI scripts to see what's going on.
